sdk/communication/azure-communication-identity/tests/test_communication_identity_client.py

```diff
@@ -110,10 +120,12 @@ def test_get_token_from_managed_identity(self):
         )
         user = identity_client.create_user()
 
-        token_response = identity_client.get_token(user, scopes=[CommunicationTokenScope.CHAT])
+        token_response = identity_client.get_token(user, scopes=scopes)
 
         assert user.properties.get('id') is not None
         assert token_response.token is not None
+        # payload = jwt.decode(token_response.token, options={"verify_signature": False})
```

Comment:
these comments should be here?

Reply:
Then we can also remove import of the [jwt](https://github.com/Azure/azure-sdk-for-python/blob/0502b772e0352614a9247cbe2f10a0384a148176/sdk/communication/azure-communication-identity/tests/test_communication_identity_client.py#L9)

Reply:
Sure